src/intTest/resources/fhir/multi_pathology_msg1.fhir.json

```diff
@@ -89,10 +154,22 @@
             }
         },
         {
-            "fullUrl": "urn:uuid:b019bdf7-53fe-42a6-89f4-24a6ab5be2c4",
+            "fullUrl": "urn:uuid:df8c2de0-c846-4041-a2e9-5cdf1e6390aa",
+            "resource": {
+                "resourceType": "Practitioner",
```

Comment:
So now we create a requester or performer Practitioner resource even if there is no information in EDIFACT